webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrengthSelectors.js

```diff
@@ -0,0 +1,2 @@
+export const doesPasswordsMatch = ({ password, passwordConfirmation }) =>
```

Comment:
Yes, I believe we should adopt this pattern but I wouldn't run to migrating, mostly for new features.

Doesn't the redux-reselect syntax feel a bit confusing?

Reply:
was actually going to suggest some kind of memoized selector as well (after seeing that redux-reselect) previously... not much code there: https://github.com/reactjs/reselect/blob/master/src/index.js

Reply:
@priley86 @amirfefer Now when I understand that `reselect` act as a memoized selector and I read more about it, it makes a lot of sense to use it (because it's memorized).

But, don't you think it takes a very simple concept like selectors and makes it really complicated to write and understand? 
especially for developers how new to the concept of redux.

Reply:
it is just minor performance gain in many cases. I think we abstract that method into a common helper (either reselect or just write our own). Then it becomes less about beginner knowledge and just a function. Personal preference though...